tests/ctst/features/azureArchive.feature

```diff
@@ -122,3 +122,28 @@ Feature: Azure Archive
         |               Versioned |           2 |        100 |           2 |
         # |               Suspended |           2 |        100 |           2 |
 
+    @2.7.0
+    @PreMerge
+    @Flaky
```

Comment:
I know other tests are are also flagged as flaky, but do we know why they are flaky? can we make them more robust?

Reply:
The transition to cold is flaky.. I think we can make them more robust but I thinks this better goes in a separate ticket

Though, it would be nice to set a step as flaky instead of the full test, actually only the part where we wait for the storage class to change here is flaky...
It would be clearer to have only this step as flaky